### ISSUE TITLE
Data Availability and References in Paper

### ISSUE THREAD

**Issue by rduesing** [TaymAlsalti/MRP_Norming#1]
Hello!

Many thanks for your paper and the demonstration. Nonetheless, I tried to run your code you provided on OSF/GitHub, but unfortunaltely, you are using data which lie in your folder ".../unshareable_data/..." and which I cannot find anywhere. This appears in the 01_data_cleaning.Rmd, as well as the 02_tutorial.Rmd file. Do I miss something or didn't you upload these files? If the latter one is true, how am I supposed to run your examples?

The other thing is about two references that appear in the text, but are not listed in the reference list. It is Kennedy & Gelman (2021) and Gelman (2018), where the latter appears in a foot note.

Many thanks for any help!
Best,
Rainer

**Comment by rubenarslan** [comment]
Hi Rainer,

The TwinLife data are available from GESIS, we provide the relevant link in our manuscript. As a researcher at a university, you should get access easily and quickly (hours in my experience). However, Taym also provided a synthetic dataset if you don't want to go through this trouble: It's here: https://osf.io/epkfy

This circumstance is mentioned in the manuscript:

> We do not have permission to disseminate the two versions of the TwinLife dataset we used (2 and 6), these have to be requested from the Leibniz Institute for the Social Sciences directly: [search.gesis.org/research_data/ZA6701](https://search.gesis.org/research_data/ZA6701). Alternatively, you can execute the main tutorial analyses on the synthetic dataset (created using the package synthpop, [Nowok et al., 2016)](https://www.zotero.org/google-docs/?CklNnJ) we provide on our OSF project and arrive at very similar results to those reported in the main text. You can also load the .rds file for the brm model we ran on that synthetic dataset (also available on our OSF project) if you want to avoid running the model yourself. We cannot share the raw means and SDs we extracted from the CFT 20-R manual, either.

But in the code itself, the code comment that mentions the synthetic data is placed rather late, so easy to miss. We'll change that.

Unfortunately, we did not get permission from Hogrefe to share the norm values we extracted from the CFT-20-R manual. The code should still run until the relevant section, but maybe we should remove this part from the tutorial Rmd, as we cannot make it reproducible.

Thank you for noticing the missing references and telling us about them! We'll correct this too. We're really happy that someone is trying out the code and would be grateful if you let us know if you encounter more hurdles. 

PS.: I now [owe](https://rubenarslan.github.io/bug_bounty.html) you a beer and a tenner :-)

**Comment by rduesing** [comment]
Dear Ruben, 

many, many thanks for the quick reply. ;-)  As it seems, I missed the last paragraph in your article, sorry for that. I am a kind of "hands on" guy, who tries to understand the code so I started with it parallel to reading the article. As you wrote, the comment in the script comes pretty late and maybe it would be helpful to make it more clear that you can run the simulated data alternatively, but you do not need the data wrangling at the beginning, and take the simulated data as they are. 

I will open an issue, if I encounter anything else.

All the best, 
Rainer

P.S.: Hope to see you on a conference for the beer and the tenner ;-) THX

**Comment by TaymAlsalti** [comment]
Dear Rainer,

Thanks again from my side, too! I reorganised the code and added additional information to make it clear which parts of our code are directly reproducible and which aren't. I also updated the readme. 

Besides the beer and the tenner, Twitter might be a sinking ship, but maybe you'll consider joining to claim the glory I spoke in your name anyway? https://twitter.com/TaymAlsalti/status/1720008100114018658

Best,
Taym
